src/estimators/tests/stratified_sketch_test.py

```diff
@@ -0,0 +1,170 @@
+# Copyright 2020 The Private Cardinality Estimation Framework Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for wfa_cardinality_estimation_evaluation_framework.estimators.stratified_sketch."""
+
+from absl import logging
+from absl.testing import absltest
+from absl.testing import parameterized
+
+import farmhash
+import numpy as np
+
+from wfa_cardinality_estimation_evaluation_framework.estimators import stratified_sketch
+from wfa_cardinality_estimation_evaluation_framework.estimators.exact_set import ExactMultiSet
+from wfa_cardinality_estimation_evaluation_framework.simulations import set_generator
+
+ONE_PLUS = '1+'
```

Comment:
It's better to refer to the definition of ONE_PLUS in stratified_sketch.py than to redefine it here.

Reply:
definetly, thanks!